read-fonts/src/tables/variations.rs

```diff
@@ -390,24 +390,28 @@ impl<'a> ExactSizeIterator for PackedPointNumbersIter<'a> {}
 #[derive(Clone, Debug)]
 pub struct PackedDeltas<'a> {
     data: FontData<'a>,
+    // How many values we expect
     count: usize,
 }
 
 impl<'a> PackedDeltas<'a> {
+    pub fn new(data: FontData<'a>, count: usize) -> Self {
```

Comment:
does this need to be pub-pub? could it be pub(crate)? If pub, are there any invariants we should document?

Reply:
Good catch, updated to `pub(crate)`